### PR TITLE
User edit

### PR DESCRIPTION
**Pull request by marchingkoala**
- added extra security in App component so that only logged in users can view & edit their account
- account page was set up with random profile photo & edit button
- account page's saved recipe section will be added later with separate ticket
- user edit page component added
- form to edit user's information has been added

### CODE REVIEW

File: src/components/User/UserEdit.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    const id = user.id;
+    const [first_name, setfirst_name] = useState(user.first_name);
+    const [last_name, setLast_name] = useState(user.last_name);
+    const [phoneNumber, setPhoneNumber] = useState(user.phoneNumber);
+
+    const firstnameHandler = (event: any) => {
+        setfirst_name(event.target.value);
+    }
+    const lastnameHandler = (event: any) => {
+        setfirst_name(event.target.value);
```
[the remark this answers is not in the document]
Comment (marchingkoala) [reply src/components/User/UserEdit.tsx]: resolved

File: src/components/App.tsx [hunk 2]
```diff
@@ -71,13 +72,22 @@ const App = () => {
           <nav className="navbar">
             <Link to="/">Home</Link>
             <Link to="/about">About</Link>
-            <Link to="/user">Account</Link>
+            {/*only logged-in user can view below tabs */}
+            {user.id && (
+              <>
+              <Link to="/user">Account</Link>
+              <Link to="/ingredient">Fridge</Link>
```
[the remark this answers is not in the document]
Comment (marchingkoala) [reply src/components/App.tsx:79]: resolved

File: src/components/User/UserEdit.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    }
+    const phoneNumberHandler = (event: any) => {
+        setfirst_name(event.target.value);
```
[the remark this answers is not in the document]
Comment (marchingkoala) [reply src/components/User/UserEdit.tsx]: resolved

**Review by emccormick16 — APPROVED** [review]
Just need to fix the logout on submitting changes

**Pull request merged**